### PR TITLE
Remove responding to json from controllers

### LINKED ISSUE #206: Stop responding to json
A lot of the controllers currently respond to both html and json. We don't need them to respond to json, this is a leftover from rails generators.

**Solution**
From the respond_to blocks we only need the part that is responding to html. Extract that html part and erase the rest of it. Here's an example of one of many controllers that leverage the respond_to block without needing it.

This is what we currently have:

```ruby
def update
  authorize @membership_type

  respond_to do |format|
    if @membership_type.update(membership_type_params)
      format.html { redirect_to @membership_type, notice: 'Membership type was successfully updated.' }
      format.json { render :show, status: :ok, location: @membership_type }
    else
      format.html { render :edit }
      format.json { render json: @membership_type.errors, status: :unprocessable_entity }
    end
  end
end
```

This is where we wanna be:

```ruby
def update
  authorize @membership_type

  if @membership_type.update(membership_type_params)
    redirect_to @membership_type, notice: 'Membership type was successfully updated.'
  else
    render :edit
  end
end
```

**Additional context**
Please find all of the occurrences of this behavior and remove it from our controllers.

### PR DESCRIPTION
**Pull request by mauricio-prl**
Resolves https://github.com/rubyforgood/babywearing/issues/206

### Description
This PR is just removing the json responding from controllers.

### CODE REVIEW

File: app/controllers/categories_controller.rb [hunk 1]
```diff
@@ -24,46 +22,33 @@ def edit
...
   # DELETE /categories/1
-  # DELETE /categories/1.json
   def destroy
     authorize @category
 
-    @category.destroy
-    respond_to do |format|
-      format.html { redirect_to categories_url, alert: 'Category was successfully destroyed.' }
-      format.json { head :no_content }
+    if @category.destroy
```
[the remark this answers is not in the document]
Comment (mauricio-prl) [reply app/controllers/categories_controller.rb:50]: Can it be ignored?

[the remark this answers is not in the document]
Comment (DamirSvrtan) [reply app/controllers/categories_controller.rb:50]: I'll check it out, I agree, this currently reads better!

[the remark this answers is not in the document]
Comment (DamirSvrtan) [reply app/controllers/categories_controller.rb:50]: Hi @m-pereira, can you put this line in the .rubocop.yml file:

```
Style/IfUnlessModifier:
  Enabled: false
```

(right below Rails enabled true).

**Review by DamirSvrtan — CHANGES_REQUESTED** [review]
Hey @m-pereira, thanks for another contribution! 🎉 Can you please add that rubocop line and as well simplify  the [signed agreements controller](https://github.com/rubyforgood/babywearing/blob/master/app/controllers/signed_agreements_controller.rb#L13) (remove the respond_to).

File: app/controllers/categories_controller.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (mauricio-prl) [reply app/controllers/categories_controller.rb:50]: It's added

**Comment by m-pereira** [comment]
@DamirSvrtan I saw that there are many rubocop offenses, I'm going to add a new PR to fix that!

**Comment by DamirSvrtan** [comment]
This is great @m-pereira, thanks for a great contribution! 🎉

**Pull request merged**